### PR TITLE
More detailed allOf fail message

### PR DESCRIPTION
**Pull request by slipovenko**
Added returning detailed description for the allOf validation case.
Helps to understand what exactly happened

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 author = 'Konstantin Valetov'
 
 # The full version, including alpha/beta/rc tags
-release = '0.5.3'
+release = '0.5.4'
```
Comment (hh-h) [inline docs/conf.py:25]: please, remove this change

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+0.5.4 (23-03-2021)
```
Comment (hh-h) [inline CHANGELOG.rst:4]: replace date with `dd-mm-yyyy` placeholder

File: aiohttp_swagger3/validators.py [hunk 3]
```diff
@@ -460,7 +498,7 @@ def validate(self, raw_value: Any, raw: bool) -> Any:
             try:
```
Comment (hh-h) [inline aiohttp_swagger3/validators.py:498]: the `try` block can be removed now at all

**Review by hh-h — CHANGES_REQUESTED** [review]

**Comment by hh-h** [comment]
Hi, use `tox -e format` to reformat the code.

File: tests/test_docs_request_bodies.py [hunk 4]
```diff
@@ -2,10 +2,33 @@
...
 from aiohttp import web
+from pytest import raises
+
+from aiohttp_swagger3.exceptions import DiscriminatorValidationError
+from aiohttp_swagger3.validators import Discriminator, Validator
 
 from .helpers import error_to_json
 
 
+async def test_validator_not_implemented():
```
Comment (hh-h) [inline tests/test_docs_request_bodies.py:13]: please, remove this test case, It is not related to MR.

File: tests/test_docs_request_bodies.py [hunk 5]
```diff
@@ -2,10 +2,33 @@
...
+            def t(self):
+                pass
+
+        value = ""
+        raw = {}
+        tv = TestValidator()
+        tv.validate(value, raw)
+
+
+async def test_discriminator():
```
Comment (hh-h) [inline tests/test_docs_request_bodies.py:26]: please, remove this test case, It is not related to MR.

File: tests/test_docs_request_bodies.py [hunk 6]
```diff
@@ -1138,7 +1161,6 @@ async def handler(request, body: Dict):
 
 
 async def test_object_can_have_optional_props(swagger_docs, aiohttp_client):
-
```
Comment (hh-h) [inline tests/test_docs_request_bodies.py:1141]: leave it as is, please

File: tests/test_docs_request_bodies.py [hunk 7]
```diff
@@ -1315,7 +1337,6 @@ async def handler(request, body: Dict):
 
 
 async def test_nullable_ref(swagger_docs_with_components, aiohttp_client):
-
```
Comment (hh-h) [inline tests/test_docs_request_bodies.py:1318]: leave it as is, please

**Review by hh-h — CHANGES_REQUESTED** [review]

**Comment by slipovenko** [comment]
@hh-h Hi
Guess 0.01 test coverage reduce is not that critical for merging.

File: tests/test_docs_request_bodies.py [hunk 8]
```diff
@@ -571,12 +571,16 @@ async def handler(request, body: Dict):
                         properties:
                           id:
                             type: integer
+                          rank:
```
Comment (hh-h) [inline tests/test_docs_request_bodies.py:574]: You should not touch anything that not related to your functionality. This function tests `oneOf`, your change is about `allOf`

**Review by hh-h — CHANGES_REQUESTED** [review]
> @hh-h Hi
> Guess 0.01 test coverage reduce is not that critical for merging.

That's fine.

File: tests/test_docs_request_bodies.py (hunk 8 above)
Comment (slipovenko) [reply tests/test_docs_request_bodies.py:574]: oh. sorry

**Review by hh-h — APPROVED** [review]

**Comment by slipovenko** [comment]
Will you make it available via PyPi please?

**Comment by hh-h** [comment]
I'll release the new version on the weekend.

**Comment by hh-h** [comment]
@slipovenko released 0.5.4 https://pypi.org/project/aiohttp-swagger3/

**Pull request merged**